### PR TITLE
AYR-671 - Record Arrangement

### PR DESCRIPTION
**Pull request by colinbowen**
## Changes in this PR
- add styles for record arangment
- update tests

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-671

### After
Screenshot 2024-03-12 at 16 45 28

Screenshot 2024-03-12 at 16 45 47

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/static/src/scss/includes/_record.scss
Comment (anthonyhashemi) [inline app/static/src/scss/includes/_record.scss]: apparently `calc(var(--border-width) * -1);` should work

**Review by anthonyhashemi — DISMISSED** [review]

File: app/static/src/scss/includes/_record.scss
Comment (colinbowen) [reply app/static/src/scss/includes/_record.scss]: Added this change, thanks

**Comment by pgandhizaizi** [comment]
looks good, please wait until Candace PR AYR-785 get done and merged to main

**Comment by anthonyhashemi** [comment]
will reapprove after merge conflicts fixed

**Review by anthonyhashemi — APPROVED** [review]

**Pull request merged**
